direct/ingest-server/src/main/java/cz/o2/proxima/server/ReplicationController.java

```diff
@@ -212,7 +212,8 @@ private void consumeLog(
     final Set<AttributeDescriptor<?>> allowedAttributes =
         new HashSet<>(replicaFamily.getAttributes());
 
-    final String name = "consumer-" + replicaFamily.getDesc().getName();
+    final String name =
+        replicaFamily.getDesc().getConsumerNameGenerator().getConsumerName("consumer-");
```

Comment:
It would be better to let the prefix be decided by the generator. I'd suggest only `replicaFamily.getDesc().getConsumerNameGenerator().apply()`

Reply:
Changed